data/Credential-Profile/AnonCreds-indy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../../viewer/src/schemas/Credential-Profile.json",
+  "Credential Profile is commonly called": "AnonCreds with did:indy",
+  "Credential Profile Description": "Well-known default profile in all Hyperledger Indy implementations. A favoured profile due to selective disclosure and predicate capability as well as privacy-preserving revocation mechanism. Standardization in community spec in progress (July 2022).",
+  "Credential Format": "AnonCred",
+  "Signing Algorithm": "CL",
+  "Revocation Algorithm": "Indy Revocation",
+  "Key Management (Issuer)": "did:indy",
+  "Key Management (Holder)": "link secrets",
+  "Trust Management": "Verifier knows Issuers",
+  "Formal Specification": "https://hyperledger.github.io/anoncreds-spec/",
+  "Implementations": "Hyperledger Indy + Aries, Esatus, iGrant Data Wallet, British Columbia CANDY profile https://github.com/ICCS-ISAC/Candy"
```

Comment:
candy is pan-canadian indy ledger implementation, current participants include more than just BC. I asked feedback around the discussion we had and it was concluded not a profile but an indy implementation. There is no formal specification like there is for the ebsi. I would consider it to fall under the Hyperledger Indy implementation otherwise I could see a scope creep with the amount of other indy ledgers out in the wild (sovrin, bcovrin & indicio). Listing independent wallets could also result in a very sizeable list. 

Other implementations I know of include:
Wallets: 
Northern Block Orbit Edge, Trinsic, BC wallet (Aries Bifold), Lissi wallet, 2060.io, Indicio Holdr

Platforms:
paradym.id from Animo, 2060.io, Trinsic, Indicio Proven

Codebase:
anoncreds-rs <- including this seems worthwhile